sdks/python/apache_beam/runners/dataflow/dataflow_runner_test.py

```diff
@@ -594,6 +594,121 @@ def test_combine_values_translation(self):
     self.assertIn(
         u'CombineValues', set(step[u'kind'] for step in job_dict[u'steps']))
 
+  def test_read_create_translation(self):
```

Comment:
These three feel very redundant; can a common helper method be used here?

Reply:
Done, moved to a single expect method.